src/dotty/tools/dotc/core/TypeApplications.scala

```diff
@@ -44,7 +45,7 @@ class TypeApplications(val self: Type) extends AnyVal {
   /** The type parameters of this type are:
    *  For a ClassInfo type, the type parameters of its class.
    *  For a typeref referring to a class, the type parameters of the class.
-   *  For a typeref referring to an alias or abstract type, the type parameters of
+   *  For a typeref referring to a Lambda class, the type parameters of
```

Comment:
Shouldn't this be `For a typeref referring to an alias type or a Lambda class, ...` ?